src/components/List/List.tsx

```diff
@@ -67,7 +67,7 @@ const List: React.FC<listProps> = ({ listItems, columns }) => {
             next
           </button>
           <br />
-          showing 10 out of {listItems.length} restaurants
+          showing 10 out of {listItems.length} items
```

Comment:
I realized that if the `List` is intended for re-use it shouldn't have "restaurants" hard coded. If I continue to push the capabilities of these Components, it'd be nice to create a way for the programmer to tell the List what to call the data it's displaying.

I'd like to also work towards the pagination looking like a more typical search engine, with more than the current page displayed, something like: `⏪ 1, 2 3 ... ⏩`